app/services/migration/migrators/course.rb

```diff
@@ -1,11 +1,29 @@
 module Migration::Migrators
   class Course < Base
+    COURSE_GROUPS = {
+      "npq-ehco-june" => "ehco",
+      "npq-specialist-autumn" => "specialist",
+      "npq-leadership-autumn" => "leadership",
+      "npq-ehco-march" => "ehco",
+      "npq-leadership-spring" => "leadership",
+      "npq-ehco-december" => "ehco",
+      "npq-specialist-spring" => "specialist",
+      "npq-ehco-november" => "ehco",
+      "npq-aso-november" => "support",
+      "npq-aso-december" => "support",
+      "npq-aso-march" => "support",
+      "npq-aso-june" => "support",
+    }.freeze
+
     def call
       migrate(ecf_npq_courses, :course) do |ecf_npq_course|
+        course_group = ::CourseGroup.find_or_initialize_by(name: COURSE_GROUPS[ecf_npq_course.identifier])
```

Comment:
I think those are not course identifiers but schedules unless I'm mistaken?

Reply:
Ah yes. I'll remove this for now.